mcp-tools/maos-mcp-hub/gateways/jira/actions.py

```
@@ -0,0 +1,335 @@
+    client = get_client()
+    fields: Dict[str, Any] = {}
+    if description:
+        fields["description"] = {"type": "doc", "version": 1, "content": [
+            {"type": "paragraph", "content": [{"type": "text", "text": description}]}
+        ]}
+    if labels:
+        fields["labels"] = labels
+    return await client.create_issue(project_key, issue_type, summary, **fields)
+
+
+async def edit_issue(issue_key: str, **fields) -> dict:
```

Ruff (0.15.9)

[warning] Missing type annotation for `**fields`

(ANN003)